lib/drives/pages/drive_detail_page.dart

```diff
@@ -394,6 +322,11 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
       ),
     );
   }
+
+  void _cancelDrive() async {
+    await _drive?.cancel();
+    setState(() {});
```

Comment:
can we add loadDrive here, so that the page is reloaded, also when the drive is cancelled the button to cancel the drive should either be removed or changed to reactivate Drive or something.

Reply:
That should actually happen automatically (it does in the ride view) because of the setState

There actually was a bug that was preventing it, so thanks for catching that 👍